module/Application/Module.php

```diff
@@ -111,7 +111,7 @@ public function onRender(MvcEvent $e)
         if (!$e->getViewModel()->terminate()) {
             $entityManager = $this->services->get('Doctrine\ORM\EntityManager');
             $e->getViewModel()
-              ->setVariable('modules_list', $entityManager->getRepository('Application\Entity\ModuleList')->findAll());
+              ->setVariable('modules_list', $entityManager->getRepository('Application\Entity\ModuleList')->findBy([], ['id' => 'DESC']));
```

Comment:
for `module_list` in left sidebar on each module, I'm thinking we should use `random()` with limited modules shown. Thoughts?